.travis.yml

```diff
@@ -29,11 +35,19 @@ script:
   # Build maestro
   - ./build.sh
 
+  # Run devicedb in docker
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      docker run --rm -d -p 127.0.0.1:9090:9090/tcp senramakri/test_images:devicedb_nocluster
+      sleep 10
```

Comment:
I take it this line waits for devicedb to be brought up? Is there no cleaner /definitive way to detect when the service is ready?

Reply:
Usually it doesnt take that long, if it didnt start the tests will fail and will clearly say it cant connect. So we can detect that easily from logs. Anyway, let me know if we need a definite way.